docs/guide/advanced/vuex.md

```diff
@@ -181,6 +181,48 @@ By creating a `createVuexStore` function that takes an initial state, we can eas
 
 The [Vue Testing Handbook](https://lmiller1990.github.io/vue-testing-handbook/testing-vuex.html) has more examples for testing Vuex. Note: the examples pertain to Vue.js 2 and Vue Test Utils v1. The ideas and concepts are the same, and the Vue Testing Handbook will be updated for Vue.js 3 and Vue Test Utils 2 in the near future.
 
+## Testing a store when using the composition API
+When using the composition API, the Vuex store is usually accessed via a `useStore`-helper function. In Javascript projects the built-in `useStore` function should be used. More about it here: https://next.vuex.vuejs.org/guide/composition-api.html.
```

Comment:
```suggestion
When using the Composition API, a Vuex store is usually accessed via a `useStore` helper function. In JavaScript projects the built-in `useStore` function should be used. More about it here: https://next.vuex.vuejs.org/guide/composition-api.html.
```

Reply:
I don't understand what "In JavaScript projects the built-in `useStore` function should be used" means. Isn't this the same information as the first sentence?

Reply:
Okay, I removed it.